runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -170,9 +171,9 @@ public FastObjectLoader addStreamsToLoad(List<String> streamsToLoad) {
     /**
      * We can register streams with non-default type
      */
-    private final Map<UUID, ObjectBuilder> customTypeStreams = new HashMap<>();
+    private final Map<UUID, Builder> customTypeStreams = new HashMap<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651447)